src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -67,6 +69,12 @@ public void delete(Long id) {
         lineRepository.delete(line);
     }
 
+    public void deleteSection(Long lineId, Long stationId) {
+        Line line = lineRepository.findById(lineId)
+                                  .orElseThrow(NoResultDataException::new);
+        line.removeSection(stationService.findByIdThrow(stationId));
```

Comment:
개인적으로 퍼블릭 메서드 명을 정할때는 협력에 맞게 추상적인 작성하는 것을 선호하는데요.
그 이유는 내부 구현이 바뀌더라도, 메서드 명을 유지할 수 있기 때문입니다.
메서드 명을 유지한다는 것은 생각보다 단순해보이기도하지만, 
처음 정의한 협력을 바꾸지 않는다는 관점에서 바라보면 굉장히 중요하게 느껴지기도 하네요 🤔

해당 메서드에서 더 이상 예외를 던지지 않도록 변경되어야한다고 가정해보면 내부 구현이 변할때 메서드 명까지 함께 변해야 하네요!

> 구현과 관련된 모든 정보를 캡슐화하고 객체의 퍼블릭 인터페이스에는 협력과 관련된 의도만을 표현해야 한다.

```suggestion
        line.removeSection(stationService.findById(stationId));
```